### PR TITLE
README: add an example with global node config

### PR DESCRIPTION
**Pull request by gyuho**
```
avalanche-network-runner server \
--log-level debug \
--port=":8080" \
--grpc-gateway-port=":8081"
```

> `curl -X POST -k http://localhost:8081/v1/control/start -d '{"execPath":"'${AVALANCHEGO_EXEC_PATH}'","globalNodeConfig":"{\"http-host\":\"0.0.0.0\"}","numNodes":5,"logLevel":"INFO","pluginDir":"'${AVALANCHEGO_PLUGIN_PATH}'","customVms":{"subnetevm":"/tmp/subnet-evm.genesis.json"}}'`

Screen Shot 2022-07-08 at 5 42 22 PM

> `{"level":"info","ts":"2022-07-08T17:34:02.463+0800","caller":"server/server.go:315","msg":"starting","execPath":"/Users/leegyuho/go/src/github.com/ava-labs/avalanchego/build/avalanchego","numNodes":5,"whitelistedSubnets":"","pid":8114,"rootDataDir":"/var/folders/xt/szh_n5x154x2hyhj0zkpd3qr0000gn/T/network-runner-root-data_20220708_173402","pluginDir":"/Users/leegyuho/go/src/github.com/ava-labs/avalanchego/build/plugins","chainConfigs":null,"defaultNodeConfig":"{\"http-host\":\"0.0.0.0\"}"}`

Signed-off-by: Gyuho Lee <gyuho.lee@avalabs.org>

### CODE REVIEW

**Comment by gyuho** [comment]
e.g.,

```
# from avalanche-network-runner server logs
# to find the log for custom VM plugin binary
{"level":"info","ts":"2022-07-12T12:38:12.166+0800","caller":"server/custom_vms.go:289","msg":"inspecting node log directory for custom VM logs","node-name":"node1","log-dir":"/var/folders/xt/szh_n5x154x2hyhj0zkpd3qr0000gn/T/network-runner-root-data_20220712_123646/node1/log"}
{"level":"info","ts":"2022-07-12T12:38:12.166+0800","caller":"server/custom_vms.go:295","msg":"checking log","vm-id":"srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy","subnet-id":"p433wpuXyJiDhyazPYyZMJeaoPSW76CBZ2x7wrVPLgvokotXz","blockchain-id":"RtoUnT4mLXZLkWNYDwFB8w7rNsev3VY1BgpQxJWqJZG6EwJ1N","log-path":"/var/folders/xt/szh_n5x154x2hyhj0zkpd3qr0000gn/T/network-runner-root-data_20220712_123646/node1/log/RtoUnT4mLXZLkWNYDwFB8w7rNsev3VY1BgpQxJWqJZG6EwJ1N.log"}
{"level":"info","ts":"2022-07-12T12:38:12.166+0800","caller":"server/custom_vms.go:304","msg":"found the log","log-path":"/var/folders/xt/szh_n5x154x2hyhj0zkpd3qr0000gn/T/network-runner-root-data_20220712_123646/node1/log/RtoUnT4mLXZLkWNYDwFB8w7rNsev3VY1BgpQxJWqJZG6EwJ1N.log"}
{"level":"info","ts":"2022-07-12T12:38:12.166+0800","caller":"server/custom_vms.go:289","msg":"inspecting node log directory for custom VM logs","node-name":"node2","log-dir":"/var/folders/xt/szh_n5x154x2hyhj0zkpd3qr0000gn/T/network-runner-root-data_20220712_123646/node2/log"}

# to see the log files
find /var/folders/xt/szh_n5x154x2hyhj0zkpd3qr0000gn/T/network-runner-root-data_20220712_123646/node1/log
/var/folders/xt/szh_n5x154x2hyhj0zkpd3qr0000gn/T/network-runner-root-data_20220712_123646/node1/log
/var/folders/xt/szh_n5x154x2hyhj0zkpd3qr0000gn/T/network-runner-root-data_20220712_123646/node1/log/RtoUnT4mLXZLkWNYDwFB8w7rNsev3VY1BgpQxJWqJZG6EwJ1N.log
/var/folders/xt/szh_n5x154x2hyhj0zkpd3qr0000gn/T/network-runner-root-data_20220712_123646/node1/log/C.log
/var/folders/xt/szh_n5x154x2hyhj0zkpd3qr0000gn/T/network-runner-root-data_20220712_123646/node1/log/P.log
/var/folders/xt/szh_n5x154x2hyhj0zkpd3qr0000gn/T/network-runner-root-data_20220712_123646/node1/log/main.log
/var/folders/xt/szh_n5x154x2hyhj0zkpd3qr0000gn/T/network-runner-root-data_20220712_123646/node1/log/X.log

# to read log file
tail /var/folders/xt/szh_n5x154x2hyhj0zkpd3qr0000gn/T/network-runner-root-data_20220712_123646/node1/log/RtoUnT4mLXZLkWNYDwFB8w7rNsev3VY1BgpQxJWqJZG6EwJ1N.log
WARN [07-12|12:37:54.116] <RtoUnT4mLXZLkWNYDwFB8w7rNsev3VY1BgpQxJWqJZG6EwJ1N Chain> github.com/ava-labs/subnet-evm/core/rawdb/accessors_metadata.go:111: Error reading unclean shutdown markers error="not found"
INFO [07-12|12:37:54.116] <RtoUnT4mLXZLkWNYDwFB8w7rNsev3VY1BgpQxJWqJZG6EwJ1N Chain> github.com/ava-labs/subnet-evm/core/tx_pool.go:505: Transaction pool price threshold updated price=0
[07-12|12:37:54.116] INFO <RtoUnT4mLXZLkWNYDwFB8w7rNsev3VY1BgpQxJWqJZG6EwJ1N Chain> proposervm/vm.go:304 block height index was successfully verified
[07-12|12:37:54.117] INFO <RtoUnT4mLXZLkWNYDwFB8w7rNsev3VY1BgpQxJWqJZG6EwJ1N Chain> snowman/transitive.go:69 initializing consensus engine
[07-12|12:37:54.117] INFO <RtoUnT4mLXZLkWNYDwFB8w7rNsev3VY1BgpQxJWqJZG6EwJ1N Chain> bootstrap/bootstrapper.go:112 Starting bootstrap...
[07-12|12:37:54.117] INFO <RtoUnT4mLXZLkWNYDwFB8w7rNsev3VY1BgpQxJWqJZG6EwJ1N Chain> common/bootstrapper.go:282 Bootstrapping skipped due to no provided bootstraps
[07-12|12:37:54.117] INFO <RtoUnT4mLXZLkWNYDwFB8w7rNsev3VY1BgpQxJWqJZG6EwJ1N Chain> bootstrap/bootstrapper.go:493 bootstrapping fetched 0 blocks. Executing state transitions...
[07-12|12:37:54.117] INFO <RtoUnT4mLXZLkWNYDwFB8w7rNsev3VY1BgpQxJWqJZG6EwJ1N Chain> queue/jobs.go:182 executed 0 operations
[07-12|12:37:54.118] INFO <RtoUnT4mLXZLkWNYDwFB8w7rNsev3VY1BgpQxJWqJZG6EwJ1N Chain> snowman/transitive.go:343 consensus starting with MRYPdsTQGrQdUpiJUkzJR6uo9wXqh9q1ZfsaSkpytd171CELr as the last accepted block
INFO [07-12|12:37:54.119] <RtoUnT4mLXZLkWNYDwFB8w7rNsev3VY1BgpQxJWqJZG6EwJ1N Chain> github.com/ava-labs/subnet-evm/plugin/evm/vm.go:629: Enabled APIs: public-eth, public-eth-filter, net, web3, internal-public-eth, internal-public-blockchain, internal-public-transaction-pool
```

File: README.md [hunk 1]
```diff
@@ -496,6 +496,10 @@ AVALANCHEGO_PLUGIN_PATH="${HOME}/go/src/github.com/ava-labs/avalanchego/build/pl
 
 curl -X POST -k http://localhost:8081/v1/control/start -d '{"execPath":"'${AVALANCHEGO_EXEC_PATH}'","numNodes":5,"logLevel":"INFO","pluginDir":"'${AVALANCHEGO_PLUGIN_PATH}'","customVms":{"subnetevm":"/tmp/subnet-evm.genesis.json"}}'
 
+# or to expose local RPC server to all traffic
```
Comment (felipemadero) [inline README.md:499]: can we made the example but avoiding the custom VM options. just the minimal options for the desired behaviour

**Review by felipemadero — COMMENTED** [review]
besides the comment. LGTM. will approve after it.

**Comment by dasconnor** [comment]
Did this actually work? Those logs say the rpc is running on 127.0.0.1

**Comment by felipemadero** [comment]
> Did this actually work? Those logs say the rpc is running on 127.0.0.1

There is an issue with server RPC endpoints, it uses have constant 127.0.0.1. We may change that to return RPC endpoints 0.0.0.0 based, if needed.

**Comment by felipemadero** [comment]
> > Did this actually work? Those logs say the rpc is running on 127.0.0.1
> 
> There is an issue with server RPC endpoints, it uses have constant 127.0.0.1. We may change that to return RPC endpoints 0.0.0.0 based, if needed.

https://ava-labs.atlassian.net/browse/AV-1902

**Review by felipemadero — APPROVED** [review]

**Pull request merged**